services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -219,6 +220,18 @@ async def create_computation(  # noqa: C901, PLR0912
                     detail=f"Project {computation.project_id} has no computational services",
                 )
 
+            if computation.wallet_info:
+                resource_usage_api = ResourceUsageApi.get_from_state(request.app)
+                # NOTE: MD/SAN -> add real service version/key when it is more clear how we will proceed
```

Comment:
here, you cannot do that. a computation contains many services, that means this call must be done for every service inside the run. I can show you where that is.

Reply:
Yes, we can sync today :+1: (my idea was that this should have been as an intermediary step before the final version, but we can discuss)